Motivations.md

```diff
@@ -132,7 +132,7 @@ RSocket supports session resumption, allowing a simple handshake to resume a cli
 
 RSocket supports two forms of application-level flow control to help protect both client and server resources from being overwhelmed.
 
-This protocol is designed for use both in datacenter, server-to-server, use cases, as well as server-to-device use cases over the internet, such as to mobile devices or browsers. 
+This protocol is designed to handle the server-to-server (in-datacenter/cloud) use case, and the device-to-server ([edge network](https://www.omnisci.com/technical-glossary/edge-network)) use case. As well as server-to-device use cases over the internet, such as to mobile devices or browsers.
```

Comment:
I like the change, but the combination of both device-to-server and server-to-device reference is new and also awkward.  Most likely devices connect to server but allows bi-di comms.

Reply:
should I use examples instead of the device-to-server and server-to-device, so we just remove the server-to-server and the device-to-server, and put examples of these things instead.
and then clear out that these are called device-to-server and server-to-device ?

Reply:
I think generally we have server-server (in DC, all owned by same dev), server-server (hitting other services you don't own) and client-server (mobile use case).